pytest_girder/pytest_girder/utils.py

```diff
@@ -1,10 +1,115 @@
+import asyncore
 import base64
-import json
-
 import cherrypy
+import email
+import json
+import os
 import six
+import smtpd
+import sys
+import threading
+import time
+
 from six import BytesIO
-from six.moves import urllib
+from six.moves import queue, range, urllib
+
+_startPort = 31000
+_maxTries = 100
+
+
+class MockSmtpServer(smtpd.SMTPServer):
```

Comment:
I had a random stash or branch somewhere (that I can't find now) that was almost able to remove this in favor of [pytest-localserver](https://pypi.python.org/pypi/pytest-localserver/). It's probably out of scope for this, but we should investigate it at some point.

Reply:
That would be awesome, it does claim:

> provides a threaded instance of smtpd.SMTPServer runnning on localhost.

I can look into that in a future PR if you can't find your code.